nixos/modules/services/web-apps/collabora-online.nix

```diff
@@ -0,0 +1,1638 @@
+      # set either of them. By default they are disabled. To turn the support on, please set "enabled" property to true. It works with self hosted or cloud services, free
+      # and premium as well. The "base_url" may be https://api.languagetoolplus.com/v2 if the cloud version of LanguageTool is used. Please note that your data in the
+      # document e.g. the text part of it will be sent to the cloud API. Please read the respective privacy policies, e.g. https://languagetool.org/legal/privacy.
+      languagetool = {
+        enabled = lib.mkEnableOption ''Remote Spell and Grammar Checker'';
+
+        base_url = lib.mkOption {
+          type = lib.types.str;
+          default = "";
+          description = ''HTTP endpoint for the API server, without /check or /languages postfix at the end.'';
+        };
+
+        user_name = lib.mkOption {
+          type = lib.types.str;
+          default = "";
+          description = ''LanguageTool or Duden account username for premium usage.'';
+        };
+
+        api_key = lib.mkOption {
+          type = lib.types.str;
+          default = "";
+          description = ''API key provided by LanguageTool or Duden account for premium usage.'';
+        };
+
+        ssl_verification = lib.mkOption {
+          type = lib.types.bool;
+          default = true;
+          description = ''Enable or disable SSL verification. You may have to disable it in test environments with self-signed certificates.'';
+        };
+
+        rest_protocol = lib.mkOption {
+          type = lib.types.enum [
+            ""
+            "duden"
+          ];
+          default = "";
+          description = ''REST API protocol. For LanguageTool leave it blank, for Duden Korrekturserver use the string 'duden'.'';
+        };
+      };
+
+      # DeepL API settings for translation service
+      deepl = {
+        enabled = lib.mkOption {
+          type = lib.types.bool;
+          default = false;
+          description = ''If true, shows translate option as a menu entry in the compact view and as an icon in the tabbed view.'';
+        };
+
+        api_url = lib.mkOption {
+          type = lib.types.str;
+          default = "";
+          description = ''URL for the API.'';
+        };
+
+        auth_key = lib.mkOption {
+          type = lib.types.str;
+          default = "";
+          description = ''Auth Key generated by your account.'';
+        };
+      };
+
+      mount_jail_tree = lib.mkOption {
+        type = lib.types.bool;
+        default = true;
+        description = ''Controls whether the systemplate and lotemplate contents are mounted or not, which is much faster than the default of linking/copying each file.'';
+      };
+
+      mount_namespaces = lib.mkOption {
+        type = lib.types.bool;
+        default = false;
+        description = ''Use mount namespaces instead of coolmount.'';
+      };
```

Comment:
Why not use them?

Reply:
Enabled by default in the updated commit.